layouts/basic/modules/Calendar/Extended/CalendarViewPostProcess.tpl

```diff
@@ -33,24 +33,24 @@
 					<div>
 						{if \AppConfig::module('Calendar', 'HIDDEN_DAYS_IN_CALENDAR_VIEW')}
 							<div class="btn-group btn-group-toggle js-switch js-switch--switchingDays c-calendar-switch" data-toggle="buttons">
-								<label class="btn btn-outline-primary c-calendar-switch__button js-switch--label-on{if $HISTORY_PARAMS['hiddenDays']} active{/if}">
-									<input type="radio" name="options" data-on-text="{\App\Language::translate('LBL_WORK_DAYS', $MODULE)}" autocomplete="off"{if $HISTORY_PARAMS['hiddenDays']} checked{/if}>
+								<label class="btn btn-outline-primary c-calendar-switch__button js-switch--label-on{if $HISTORY_PARAMS eq '' || !empty($HISTORY_PARAMS['hiddenDays'])} active{/if}">
```

Comment:
$HISTORY_PARAMS eq '' || !empty($HISTORY_PARAMS['hiddenDays']) x4
change to only one